src/main/java/edu/ucsb/cs156/happiercows/repositories/ProfitRepository.java

```diff
@@ -0,0 +1,11 @@
+package edu.ucsb.cs156.happiercows.repositories;
+
+import edu.ucsb.cs156.happiercows.entities.Profit;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProfitRepository extends CrudRepository<Profit, Long> {
+    //Iterable<Profit> findAll();
```

Comment:
What is this commented out line of code doing here?

Reply:
Apologies, I meant to take it out after testing as I found the CrudRepository already comes with findAll();, it will take it out.